_sass/_imla-nama.scss

```diff
@@ -35,6 +35,14 @@ html {
          url("type/FiraSans-Regular.woff") format("woff");
 }
 
+@counter-style urdu-abjad {
+    system: fixed;
+    symbols: '\0627 ' '\628' '\62C' '\62F' '\6C1' '\648' '\632' '\62D' '\637' '\6CC' '\6A9' '\644' '\645' '\646' '\636' '\639' '\641' '\635' '\642' '\631' '\633' '\62A' '\62B' '\62E' '\630' '\638' '\63A' '\634';
```

Comment:
I was actually not so sure about the abjad counting myself. I couldn't decide between _persian_ and _maghrebi_ so I chose _maghrebi_ and changed the _do chashmi he_ (_kaaf_ and _choti ye_ too) since I don't remember seeing _do chashmi he_ in published Urdu. Should it be changed to regular _persian-abjad_ ? Is there any reference material for use of abjads in Urdu?

Reply:
Yes, let's go with _persian-abjad_; I found [this edition of _Bang-e-Dara_](https://www.rekhta.org/ebooks/bang-e-dara-allama-iqbal-ebooks-11?lang=ur) that uses _persian-abjad_ for pages containing the frontmatter.

Reply:
So I was right about the _do chashmi he_ and _choti ye_ but which _kaaf_ should be used
![image](https://user-images.githubusercontent.com/62665768/177049941-fae4bc2e-0f43-47b5-92b1-17c7852e7771.png)
or regular ک ?

Reply:
Done in [91437ad](https://github.com/urduweb/imla/pull/17/commits/91437adf41920c8b2dc59752df13ac65ebdfb799).